hal/src/thumbv6m/usb/bus.rs

```diff
@@ -717,6 +717,15 @@ impl Inner {
         usb.ctrlb.modify(|_, w| w.detach().clear_bit());
     }
 
+    /// Enables/disables the Start Of Frame (SOF) interrupt
+    fn sof_interrupt(&self, enable: bool) {
+        if enable {
+            self.usb().intenset.write(|w| w.sof().set_bit());
+        } else {
+            self.usb().intenset.write(|w| w.sof().clear_bit());
```

Comment:
I think you might want `self.usb().intenclr.write(|w| w.sof().set_bit());` here instead. Per the datasheet, writing a zero to the `INTENSET` or `INTENCLR` registers' bits has no effect.

Reply:
Good catch, thanks!  I'll update the PR shortly.